src/starcheck.pl

```diff
@@ -578,6 +566,7 @@ sub json_obsids{
 
     $obs{$obsid}->check_monitor_commanding(\@bs, $or{$obsid});
     $obs{$obsid}->check_flick_pix_mon();
+    $obs{$obsid}->set_dynamic_mag_limits();
```

Comment:
Running it on very old loads is/should be fine.  I was doing that anyway to run the code through its paces as that is the primary goal of the regression tests.  I just wasn't sure if we wanted to see all of the old values change in the regression test output.  I'll continue with that as that plan.